### PR TITLE
GEODE-8137 - Implement loadService.

### PR DESCRIPTION
**Pull request by yozaner1324**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-modules/src/main/java/org/apache/geode/services/module/impl/JBossModuleService.java [hunk 1]
```diff
@@ -92,18 +98,20 @@ public boolean loadModule(ModuleDescriptor moduleDescriptor) {
         builder.addResourceRoot(ResourceLoaderSpec.createResourceLoaderSpec(resourceLoader));
       }
     } catch (IOException e) {
-      logger.error(e);
+      logger.error(e.getMessage());
```
Comment (kohlmu-pivotal) [inline geode-modules/src/main/java/org/apache/geode/services/module/impl/JBossModuleService.java:101]: Please print the stacktrace. so please revert back to `logger.error(e)`

File: geode-modules/src/main/java/org/apache/geode/services/module/impl/JBossModuleService.java [hunk 2]
```diff
@@ -92,18 +98,20 @@ public boolean loadModule(ModuleDescriptor moduleDescriptor) {
...
+
     ModuleSpec moduleSpec = builder.create();
     moduleLoader.addModuleSpec(moduleSpec);
 
     try {
       modules.put(moduleDescriptor.getVersionedName(),
           moduleLoader.loadModule(moduleSpec.getName()));
     } catch (ModuleLoadException e) {
-      logger.error(e);
+      logger.error(e.getMessage());
```
Comment (kohlmu-pivotal) [inline geode-modules/src/main/java/org/apache/geode/services/module/impl/JBossModuleService.java:114]: Please print the stacktrace. so please revert back to `logger.error(e)`

File: geode-modules/src/main/java/org/apache/geode/services/module/impl/JBossModuleService.java (hunk 1 above)
Comment (yozaner1324) [reply geode-modules/src/main/java/org/apache/geode/services/module/impl/JBossModuleService.java]: Updated

File: geode-modules/src/main/java/org/apache/geode/services/module/impl/JBossModuleService.java (hunk 2 above)
Comment (yozaner1324) [reply geode-modules/src/main/java/org/apache/geode/services/module/impl/JBossModuleService.java]: Updated.

**Pull request merged**